sql/src/test/java/io/crate/analyze/InsertFromValuesAnalyzerTest.java

```diff
@@ -73,105 +67,72 @@
-            .addGeneratedColumn("day", DataTypes.TIMESTAMPZ, "date_trunc('day', ts)", false)
-            .addGeneratedColumn("name", DataTypes.STRING, "concat(\"user\"['name'], 'bar')", false);
-        executorBuilder.addDocTable(generatedColumnTable);
-
-        RelationName generatedPkColumnRelationName = new RelationName(Schemas.DOC_SCHEMA_NAME, "generated_pk_column");
-        TestingTableInfo.Builder generatedPkColumnTable = new TestingTableInfo.Builder(
-            generatedPkColumnRelationName, SHARD_ROUTING)
-            .add("serial_no", DataTypes.INTEGER, null)
-            .add("product_no", DataTypes.INTEGER, null)
-            .add("color", DataTypes.STRING, null)
-            .addGeneratedColumn("id", DataTypes.INTEGER, "serial_no + 1", false)
-            .addGeneratedColumn("id2", DataTypes.INTEGER, "product_no + 1", false)
-            .addPrimaryKey("id")
-            .addPrimaryKey("id2");
-        executorBuilder.addDocTable(generatedPkColumnTable);
-
-        RelationName generatedPkAndPartedColumnRelationName =
-            new RelationName(Schemas.DOC_SCHEMA_NAME, "generated_pk_parted_column");
-        TestingTableInfo.Builder generatedPkAndPartedColumnTable = new TestingTableInfo.Builder(
-            generatedPkAndPartedColumnRelationName, SHARD_ROUTING)
-            .add("ts", DataTypes.TIMESTAMPZ, null, true)
-            .add("value", DataTypes.INTEGER, null)
-            .addGeneratedColumn("part_key__generated", DataTypes.INTEGER, "date_trunc('day', ts)", true)
-            .addPrimaryKey("value")
-            .addPrimaryKey("part_key__generated");
-        executorBuilder.addDocTable(generatedPkAndPartedColumnTable);
-
-        RelationName generatedClusteredByRelationName = new RelationName(Schemas.DOC_SCHEMA_NAME, "generated_clustered_by_column");
-        TestingTableInfo.Builder clusteredByGeneratedTable = new TestingTableInfo.Builder(
-            generatedClusteredByRelationName, SHARD_ROUTING)
-            .add("serial_no", DataTypes.INTEGER, null)
-            .add("color", DataTypes.STRING, null)
-            .addGeneratedColumn("routing_col", DataTypes.INTEGER, "serial_no + 1", false)
-            .clusteredBy("routing_col");
-        executorBuilder.addDocTable(clusteredByGeneratedTable);
-
-        RelationName generatedNestedClusteredByRelationName = new RelationName(Schemas.DOC_SCHEMA_NAME, "generated_nested_clustered_by");
-        TestingTableInfo.Builder generatedNestedClusteredByInfo = new TestingTableInfo.Builder(
-            generatedNestedClusteredByRelationName, SHARD_ROUTING)
-            .add("o",
-                ObjectType.builder().setInnerType("serial_number", DataTypes.INTEGER).build(),
-                null,
-                ColumnPolicy.DYNAMIC)
-            .addGeneratedColumn("routing_col", DataTypes.INTEGER, "o['serial_number'] + 1", false)
-            .clusteredBy("routing_col");
-        executorBuilder.addDocTable(generatedNestedClusteredByInfo);
+            .addTable(
```

Comment:
The new version of the table definition is much more readable, although the old builder style version is probably a lot easier to maintain.